jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/resources/org/jbpm/workbench/cm/client/resources/i18n/Constants.properties

```diff
@@ -21,6 +21,14 @@ CaseDetails=Case Details
 Owner=Owner
 Roles=Roles
 RoleAssignments=Role Assignments
+AssignedUsers=Assigned users
+AssignedGroups=Assigned groups
+AssignmentHelp=Intro user or groups separated by comma
```

Comment:
Added similar text in here: https://github.com/kiegroup/jbpm-wb/pull/652/files#diff-79e45c754aaa85e67034c166a87e3e1bR99